### ISSUE TITLE
Default I2C Address changed?

### ISSUE THREAD

**Issue by FoamyGuy** [adafruit/Adafruit_CircuitPython_MLX90393#46]
@ladyada 

The default I2C address used by this library is `0x0C` https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/blob/77628182cb16c570b5cc9d99310c89f64b8d823c/adafruit_mlx90393.py#L212

The new MLX9093 that I received recently has the address `0x18`. I was able to successfully test the example code in the repo and changes for PRs by using `address=0x18,`  in the initialization.  A0 and A1 jumpers are in default out of box state.

In the datasheet here: https://cdn-learn.adafruit.com/assets/assets/000/069/600/original/MLX90393-Datasheet-Melexis.pdf?1547824268 It contains a table that shows a range of possible addresses which does include one variation that would have address `0x18` if I understand correctly. 

![image](https://github.com/user-attachments/assets/717d6f2b-14f7-474e-ad14-9b94b9294cc3)

**Comment by brian-eng** [comment]
Yes, all the sensors I've gotten use `0x18` as the base address, which matches what the product page says.

https://www.adafruit.com/product/4022

> As of May 1, 2023 - Please note that based on chip shortages we are now shipping with default 'base' address of 0x18 not 0x0C

So to use this library with any of the recent chips you need to include `address = 0x18` as part of the settings you pass when initializing.

I'm not sure what the default should be, but there probably should at least  be something in the [README](https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/blob/b709a140c646fc736bedd5aadac37492f8345e5a/README.rst) about using a different address as the usage example won't work on any newer chips.

**Comment by caternuson** [comment]
Agree, probably would help to update some of the info to provide better awareness of this change. Maybe even update the example program. Shouldn't require any code changes to the library itself since it's just a matter of passing in the alt address. But currently, this is only documented as a footnote on the product page. (thanks for pointing that out @brian-eng)

**Comment by ladyada** [comment]
@FoamyGuy please update the examples
https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/blob/main/examples/mlx90393_simpletest.py
to have it check both I2C addresses
